### PR TITLE
feat: output human-readable change log

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Description

It can be hard to see the reason why a resource has been changed when the consistency test fails. The Terraform CLI renders this nicely but because we are running plan with struct so we can ignore certain changes we loose the formatted output. I investigated implementing the code from the Terraform CLI to render for humans however it was too complex. Instead, if there is a valid change I re-run the plan but in the basic mode that uses the CLI output.

#### No release required

- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other changes that don't affect Terraform code

#### Release required

- [ ] Bug fix (patch release (`x.x.X`): Change that fixes an issue and is compatible with earlier versions)
- [x] New feature (minor release (`x.X.x`): Change that adds functionality and is compatible with earlier versions)
- [ ] Breaking change (major release (`X.x.x`): Change that is likely incompatible with previous versions)

##### Release notes content

Replace this text with information that users need to know about the bug fixes, features, and breaking changes. This information helps the merger write the commit message that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Merge by using "Squash and merge".
- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author.

    The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).

### CODE REVIEW

File: testhelper/tests.go [hunk 1]
```diff
@@ -59,6 +61,9 @@ func (options *TestOptions) checkConsistency(plan *terraform.PlanStruct) {
 			changesJson = string(changesBytes)
 		}
 
+		// Run plan again to output the nice human-readable plan
+		//terraform.Plan(options.Testing, options.TerraformOptions)
```
Comment (ocofaigh) [inline testhelper/tests.go:65]: remove code comment

**Review by ocofaigh — COMMENTED** [review]
so it will output the plan in readable format only if consistency check fails - I'm fine with that

**Comment by ocofaigh** [comment]
@toddgiguere we no longer print that JSON - we just parse it for changes and print them

**Comment by daniel-butler-irl** [comment]
@ocofaigh @toddgiguere Are we ok with this one now?

**Comment by ocofaigh** [comment]
just one small open comment left https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/pull/544/files#r1182364334

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**